### PR TITLE
refactor: Adapt playground response format to new pagination

### PR DESCRIPTION
**Pull request by lukapeschke**

### CODE REVIEW

File: server/pyproject.toml [hunk 1]
```diff
@@ -50,9 +47,7 @@ sqlalchemy = "^1.4.44"
 pymysql = "^1.0.2"
 psycopg2 = "^2.9.5"
 pymongo = ">=4.2.0"
-redshift-connector = "^2.0.909"
 sqlalchemy-redshift = ">=0.8.11,<1"
-google-cloud-bigquery = {version = "^3.3.0", extras = ["pandas", "bqstorage"]}
```
Comment (Sanix-Darker) [inline server/pyproject.toml:55]: oohhh, good catch, we were not using those libs in the codebase ?

File: server/pyproject.toml [hunk 2]
```diff
@@ -25,21 +25,18 @@ quart = {version = ">=0.17,<0.19", optional = true}
...
 pytest-xdist = ">=2.5,<4.0"
 
 [tool.poetry.dev-dependencies]
 pytest-cov = "^4.0.0"
 pytest-mock = "^3.10.0"
 # required so we can use nosql_apply_parameters_to_query during tests
-toucan-connectors = { version = "^3.17.4", extras = ["awsathena"] }
+toucan-connectors = { version = "^4.0.0", extras = ["awsathena", "google_big_query", "mongo", "Redshift", "snowflake"] }
 pytest-benchmark = "^4.0.0"
 snowflake-sqlalchemy = "^1.4.4"
```
Comment (Sanix-Darker) [inline server/pyproject.toml:38]: We probably want to get rid of this one too ?

File: server/pyproject.toml (hunk 1 above)
Comment (lukapeschke) [reply server/pyproject.toml:55]: Well we're using them in  the tests, but they're pulled in by toucan-connectors anyway so :shrug:

File: server/pyproject.toml (hunk 2 above)
Comment (lukapeschke) [reply server/pyproject.toml:38]: Nope, it's used in the tests

**Pull request merged**